src/components/mapComponents/MapTiles.tsx

```diff
@@ -24,8 +24,8 @@ const MapTiles: FC<MapTilesProps> = ({
   addReportMarker,
   isCreateReportMapMode,
 }) => {
-  const color = ({approvedByEditor}: Tile): string => {
-    return approvedByEditor ? colors.colorTileAssigned : colors.colorTileAll;
+  const color = ({assignedUserName}: Tile): string => {
```

Comment:
is really assignedUserName of type Tile?

Reply:
Yes. When I've changed it started to work. 
ApprovedByEditor filed is not used anymore.